packages/core/README.md

```diff
@@ -1,7 +1,10 @@
+## Important note!
+This is a fork of official [craft.js](https://github.com/prevwong/craft.js) repo. It's meant for internal @candulabs usage and might introduce some breaking changes. Use at your own risk, for stable version and support, please checkout main repo.
+
 
 <div align="center" style={{d}}>
 <h1>craft.js</h1>
-<a href="https://www.npmjs.com/package/@craftjs/core">
+<a href="https://www.npmjs.com/package/@craftjs/coreaft-core">
```

Comment:
Please discard all changes made to markdown files.

Reply:
ah my bad I thought I removed them.